TrackletAlgorithm/VMStubMEMemoryCM.h

```diff
@@ -45,7 +45,7 @@ class VMStubMECMBase<DISK>
     // Bit sizes for VMStubMEMemoryCM fields
     kVMSMEFineZSize = 3,
     kVMSMEFinePhiSize = 3,
-    kVMSMEBendSize = 4,
+    kVMSMEBendSize = 3,
```

Comment:
The number of bits for the bend in the disk should be 4.

Reply:
Ok, that breaks things in the MP, but I'll track it down on my end.

Reply:
This actually works fine, since the change was only in the disk region.

Reply:
Seems like the number 3 came back again :-)

Reply:
Woops! Fixed it!